source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+﻿/*
+This example shows how to perform Hand-Eye calibration.
+*/
+
+using System;
+using System.IO;
+using YamlDotNet.RepresentationModel;
+
+
+using MathNet.Numerics.LinearAlgebra;
+using MathNet.Numerics.LinearAlgebra.Double;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            printHeader("This example shows conversions to/from Transformation Matrix");
+
+            // Read a transformation matrix
```

Comment:
Please check our [Sample standardization](https://zividlabs.atlassian.net/wiki/spaces/CS/pages/1115488261/Samples+standardization+-+printouts+comments) guidelines for comments and printouts. It relates to the use of present continuous tense and the decision whether to comment or print out. Please apply this for other comments and printouts.